packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -206,7 +206,7 @@ private MethodProvider BuildImplicitToBinaryContent()
                 new MethodSignature(ClientModelPlugin.Instance.TypeFactory.RequestContentApi.RequestContentType.FrameworkType.Name, null, modifiers, null, null, [model]),
                 new MethodBodyStatement[]
                 {
-                    new IfStatement(model.AsExpression.Equal(Null)) { Return(Null) },
+                    !model.Type.IsValueType ? new IfStatement(model.AsExpression.Equal(Null)) { Return(Null) } : MethodBodyStatement.Empty,
```

Comment:
any reason we didn't use the `_isStruct` field ?

Reply:
nope will change it